src/main/java/org/quiltmc/loader/impl/launch/common/MappingConfiguration.java

```diff
@@ -78,7 +80,7 @@ public TinyTree getMappings() {
 	}
 
 	public String getTargetNamespace() {
-		return QuiltLauncherBase.getLauncher().isDevelopment() ? "named" : "intermediary";
+		return QuiltLoaderImpl.INSTANCE.tryGetGameProvider().getNamespace();
```

Comment:
`tryGetGameProvider()` should be replaced with `getGameProvider()` (it has a better error message if it's not set).

Reply:
There is a chance that this gets called before the game provider is set, so I just changed it to check if it got a valid game provider or not.